Docs: Templating notifications (#60109)
grafana/grafana: docs/sources/alerting/manage-notifications/template-notifications/reference.md

---
aliases:
keywords:
  - grafana
  - alerting
  - notifications
  - templates
title: Reference
weight: 600
---

# Reference

> 

Author: I think Reference should be at the end, it's like the index of a book.

> The following reference contains all the data and template functions that are available when templating notifications.

Reviewer: ```suggestion
Here is all the data and template functions available to you for templating your notification messages.
```
